source/features/rgh-feature-descriptions.tsx

```diff
@@ -13,7 +13,7 @@ async function init(): Promise<void | false> {
 
 	const conversationsUrl = '/sindresorhus/refined-github/issues?q=' + encodeURIComponent(`"${feature.id}" sort:updated-desc`);
 
-	const commitInfoBox = (await elementReady('.hx_commit-tease, include-fragment.commit-loader'))!.parentElement!;
+	const commitInfoBox = (await elementReady('.Box-header.Details, include-fragment.commit-loader'))!.parentElement!;
```

Comment:
Should we drop `include-fragment.commit-loader`?

Reply:
> `include-fragment.commit-loader` is needed according to [`2683e30` (#3852)](https://github.com/sindresorhus/refined-github/pull/3852/commits/2683e300976cacbbb71eb219347cb82aa964616b).